utils.py

```diff
@@ -9,6 +11,21 @@ def get_map_center():
     lon = (config.MAP_END[1] + config.MAP_START[1]) / 2
     return lat, lon
 
+def get_scan_area():
+    """Returns the square kilometers for configured scan area"""
+    lat1 = config.MAP_START[0]
+    lat2 = config.MAP_END[0]
+    lon1 = config.MAP_START[1]
+    lon2 = config.MAP_END[1]
+    p1 = Point(lat1, lon1)
+    p2 = Point(lat1, lon2)
+    p3 = Point(lat1, lon1)
+    p4 = Point(lat2, lon1)
+
+    width = distance.distance(p1,p2).kilometers
+    height = distance.distance(p3,p4).kilometers
+    area = int(width * height)
+    return area;
```

Comment:
A **semicolon**?!

(sorry for late response, I found some time to process issues)